scripts/create-package/plopfile.ts

```diff
@@ -56,41 +57,34 @@ module.exports = (plop: NodePlopAPI) => {
       },
       {
         type: 'confirm',
-        name: 'hasExamples',
-        message: 'Create example scaffolding?',
+        name: 'isConverged',
+        message: 'Is this a converged package?',
         default: true,
-        when: answers => answers.target === 'react',
       },
       {
         type: 'confirm',
         name: 'publish',
-        message: answers =>
-          answers.hasExamples
-            ? 'Should the package be published right away (must be Yes for packages with example scaffolding)?'
-            : 'Should the package be published right away?',
-        default: (answers: Answers) => !!answers.hasExamples,
+        message: 'Should the package be published right away?',
+        default: false,
       },
     ],
     actions: (answers: Answers): Actions => {
       if (answers.target === 'react') answers = { hasTests: true, ...answers };
-      const { packageName, target, hasExamples, hasTests } = answers;
+      const { packageName, target, hasTests } = answers;
 
       const destination = `packages/${packageName}`;
-      const exampleRoot = `packages/react-examples`;
       const globOptions: AddManyActionConfig['globOptions'] = { dot: true };
 
       // Get derived template parameters
       const data = {
         packageNpmName: '@fluentui/' + packageName,
-        packageVersion: '9.0.0-alpha.0',
+        packageVersion: answers.isConverged ? '9.0.0-alpha.0' : '0.1.0',
```

Comment:
we're in beta now :)

```suggestion
        packageVersion: answers.isConverged ? '9.0.0-beta.0' : '0.1.0',
```